frontend/callback.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Authentication Callback</title>
+    <style>
```

Comment:
should move the styles to the css file

Reply:
I dont want to import the whole css file since it's large 🤣

Reply:
I see😅